guide/raids/amirdrassil.md

```diff
@@ -0,0 +1,445 @@
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target
+
+#### Talents:
+
+#### Lightning T30:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpdACIJBBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Fire T31:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrUSCgDIlk0oElUSCRAAAAAAKBkUSCaASLJpdACIJJEI?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Ensure to maximize your output while benefiting from {{ site.data.raid.amirdrassil.ignited_essence }}.
+* Use {{ site.data.talent.swg }} and {{ site.data.talent.gow }} to maintain uptime whilst dealing with {{ site.data.raid.amirdrassil.world_in_flames }} and {{ site.data.raid.amirdrassil.flame_waves }}.
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables before soaking {{ site.data.raid.amirdrassil.brand_of_damnation }}.
+* Use {{ site.data.talent.ag }} either for {{ site.data.raid.amirdrassil.world_in_flames }} or to remove {{ site.data.raid.amirdrassil.cauterizing_wound }}.
+* If talented, use {{ site.data.talent.earth_shield }} and {{ site.data.talent.healing_stream_totem }} as a movement filler throughout the fight.
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players dodging {{ site.data.raid.amirdrassil.world_in_flames }}.
+
+</div>
+</div>
+</div>
+
+<div class="card">
+<div class="card-header" id="tindral_sageswift">
+<div data-toggle="collapse" data-target="#tindral_sageswift-collapse" aria-expanded="true" aria-controls="tindral_sageswift-collapse" class="raid-header amirdrassil_the_dreams_hope8"><h2>Tindral Sageswift, Seer of the Flame</h2></div>
+<div class=""></div>
+</div>
+<div id="tindral_sageswift-collapse" class="collapse" aria-labelledby="tindral_sageswift" data-parent="#accordion">
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target / AoE
+
+### Talents:
+
+#### Fire:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpECIRIBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Lightning:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrIJtkgDIINKRJtkQOAAAAAAgSAJlkgmASLJpkSASkQgA?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Focus down {{ site.data.raid.amirdrassil.fiery_vines }} and {{ site.data.raid.amirdrassil.scorched_treant }} whenever they are active.
```

Comment:
I wonder if LMT is a good talent specifically for this fight and this mechanic. A small Burst AoE sounds very good (in mythic cause you just remove them in hm)?

Reply:
LMT is included in the talent suggestion for Fire T31. I guess you are referring to something else?

Reply:
No I mostly meant to turn the bullet point that way: 

- Focus down {{ site.data.raid.amirdrassil.fiery_vines }} and {{ site.data.raid.amirdrassil.scorched_treant }} whenever they are active. Try to line up your {{ site.data.talent.lmt }} with {{ site.data.raid.amirdrassil.fiery_vines }} sets. 

Or something along those lines

Reply:
Oh, I see. I like that more, too.
Going to add that!